main.nf

```diff
@@ -1,114 +1,16 @@
 #!/usr/bin/env nextflow
-
-/// To use DSL-2 will need to include this
 nextflow.enable.dsl=2
 
-// =================================================================
-// main.nf is the pipeline script for a nextflow pipeline
-// Should contain the following sections:
-	// Process definitions
-    // Channel definitions
-    // Workflow structure
-	// Workflow summary logs 
-
-// Examples are included for each section. Remove them and replace
-// with project-specific code. For more information see:
-// https://www.nextflow.io/docs/latest/index.html.
-//
-// ===================================================================
-
-// Import processes or subworkflows to be run in the workflow
-// Each of these is a separate .nf script saved in modules/ directory
-// See https://training.nextflow.io/basic_training/modules/#importing-modules 
-include { processOne } from './modules/process1'
-include { processTwo } from './modules/process2' 
-
-// Print a header for your pipeline 
-log.info """\
-
-=======================================================================================
-Name of the pipeline - nf 
-=======================================================================================
-
-Created by <YOUR NAME> 
-Find documentation @ https://sydney-informatics-hub.github.io/Nextflow_DSL2_template_guide/
-Cite this pipeline @ INSERT DOI
-
-=======================================================================================
-Workflow run parameters 
-=======================================================================================
-input       : ${params.input}
-outDir      : ${params.outDir}
-workDir     : ${workflow.workDir}
-=======================================================================================
-
-"""
-
-/// Help function 
-// This is an example of how to set out the help function that 
-// will be run if run command is incorrect or missing. 
-
-def helpMessage() {
-    log.info"""
-  Usage:  nextflow run main.nf --input <samples.tsv> 
-
-  Required Arguments:
-
-  --input	Specify full path and name of sample
-		input file (tab separated).
-
-  Optional Arguments:
-
-  --outDir	Specify path to output directory. 
-	
-""".stripIndent()
+process helloworld {
+  input: 
+    val x
+  output:
+    stdout
+  script:
+    """
+    echo '$x world!'
+    """
 }
-
-// Define workflow structure. Include some input/runtime tests here.
-// See https://www.nextflow.io/docs/latest/dsl2.html?highlight=workflow#workflow
 workflow {
-
-// Show help message if --help is run or (||) a required parameter (input) is not provided
-
-if ( params.help || params.input == false ){   
-// Invoke the help function above and exit
-	helpMessage()
-	exit 1
```

Comment:
This section will print your help message defined above. Keep it to print to the screen when you run `nextflow run main.nf --help`